preview.geoshp/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN apt-get update && apt-get install -y \
        gdal-bin \
        python-gdal \
        p7zip-full \
+    && pip install gsconfig-py3 \
```

Comment:
we should add this to the requirements.txt file.

Reply:
same for the rest of the extractors.
we probably also want to get rid of pip --upgrade pip==9.0.3 since that probably is a downgrade.